Ustring.php

```diff
@@ -220,6 +212,26 @@ public function __construct($string = null)
     }
 
     /**
+     * Check if ext/mbstring is available.
+     *
+     * @return  bool
+     */
+    public static function checkMbString()
+    {
+        return function_exists('mb_substr');
```

Comment:
Ok but from what I understand, `mbstring` is required here: as soon as the library is loaded a check is performed and an exception is thrown.